src/qtcomponents.h

```diff
@@ -15,15 +15,12 @@ class QTreeViewWithKeyEvents : public QTreeView {
     public:
         explicit QTreeViewWithKeyEvents(QWidget *parent = NULL);
```

Comment:
```suggestion
        explicit QTreeViewWithKeyEvents(QWidget *parent = nullptr);
```
and can you also please remove the `TreeViewKeyPressedEvent` since it is now unused?

Reply:
Oops, I seemed to have removed it :x

NULL replaced and `TreeViewKeyPressedEvent` removed in 84c54b8bab68e4e54965c7de2d00da0f7c775a4a